src/main/java/mate/academy/rickandmorty/dto/RickMortyDtoResponse.java

```diff
@@ -0,0 +1,15 @@
+package mate.academy.rickandmorty.dto;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Data;
+
+@Data
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class RickMortyDtoResponse {
```

Comment:
looking at class field CharacterDtoResponse is more suitable name here
```suggestion
public class CharacterDtoResponse {
```